server/datastore/mysql/software.go

```diff
@@ -315,6 +315,7 @@ SELECT
     s.release,
     s.vendor,
     s.arch,
+    s.extension_id, -- TODO: validate that this should be added
```

Comment:
@getvictor just to make sure this was meant to be added, as this SELECT returns all of the `software` columns, I'm guessing `extension_id` should be added there too.

Reply:
Yes, thank you.